cabal.project

```diff
@@ -35,33 +35,40 @@ source-repository-package
   type: git
   location: https://github.com/target/row-types
   tag: 1e8d5e084ffd46f6c7842826a1f62c60820885df
+  --sha256: 0ly5m4r8wkm8gdqyrqzsjfmp189yxsd4qp0zi3idrbgfaf45sk9k
```

Comment:
Also, how do we update these things?

Reply:
For the `cabal.project` there is a python script. For `stack.yaml` not yet. @erikd is working on something with the cardano-repo-tool I believe.